nilearn/plotting/tests/test_img_plotting.py

```diff
@@ -178,3 +184,109 @@ def test_plot_noncurrent_axes():
     for ax_name, niax in slicer.axes.items():
         ax_fh = niax.ax.get_figure()
         assert_equal(ax_fh, fh1, 'New axis %s should be in fh1.' % ax_name)
+
+
+def test_plot_connectome():
+    import pylab as pl
+    pl.switch_backend('template')
+    node_color = ['green', 'blue', 'k', 'cyan']
+    adjacency_matrix = np.array([[1., -2., 0.3, 0.],
+                                 [-2., 1, 0., 0.],
+                                 [0.3, 0., 1., 0.],
+                                 [0., 0., 0., 1.]])
+    node_coords = np.arange(3 * 4).reshape(4, 3)
+
+    args = adjacency_matrix, node_coords
+    kwargs = dict(edge_threshold=0.38,
+                  title='threshold=0.38',
+                  node_size=10, node_color=node_color)
+    plot_connectome(*args, **kwargs)
+
+    # node_coords not an array but a list of tuples
+    plot_connectome(adjacency_matrix,
+                    map(tuple, node_coords.tolist()),
+                    **kwargs)
+    # saving to file
+    with tempfile.NamedTemporaryFile(suffix='.png') as fp:
+        display = plot_connectome(*args, output_file=fp.name,
+                                  **kwargs)
+        assert_true(display is None)
+        assert_true(os.path.isfile(fp.name) and
+                    os.path.getsize(fp.name) > 0)
+
+    # with node_kwargs, edge_kwargs and edge_cmap arguments
+    plot_connectome(*args,
+                    edge_threshold='70%',
+                    title='threshold=70%',
```

Comment:
For a minor speedup, you can remove the title here: you have already tested it earlier.